ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/router-bgp-evpn.yml

```diff
@@ -43,6 +43,8 @@ router_bgp:
       allowas_in:
         enabled: true
         times: 5
+      shared_secret_profile_name: profile1
+      hash_algorithm: aes-128-cmac-96
```

Comment:
you have added secret profile in host_vars but i dont see any .cfg file generated. Can you please add a case where the secret profile applied on peer group and neighbor address

Reply:
Added test under router-bgp-evn.